src/utils.js

```diff
@@ -7,22 +7,36 @@
  * @module image/utils
  */
 
-import { widgetize, isWidget } from './widget/utils';
+import { widgetize, isWidget, setFakeSelectionLabel } from './widget/utils';
 import ModelElement from '@ckeditor/ckeditor5-engine/src/model/element';
 
 const imageSymbol = Symbol( 'isImage' );
 
 /**
  * Converts given {@link module:engine/view/element~Element} to image widget:
  * * adds {@link module:engine/view/element~Element#setCustomProperty custom property} allowing to recognize image widget element,
+ * * sets fake selection label function,
  * * calls {@link module:image/widget/utils~widgetize widgetize}.
  *
  * @param {module:engine/view/element~Element} viewElement
+ * @param {function} t Shorthand for {@link module:utils/locale~Locale#t}.
  * @returns {module:engine/view/element~Element}
  */
-export function toImageWidget( viewElement ) {
+export function toImageWidget( viewElement, t ) {
 	viewElement.setCustomProperty( imageSymbol, true );
 
+	setFakeSelectionLabel( viewElement, () => {
+		let fakeSelectionLabel = t( 'image widget' );
```

Comment:
Why isn't the label passed instead of the `t()` function? This is something which the feature should care about, not this util function.